package.json

```diff
@@ -26,7 +26,13 @@
     "build:lib": "ng build ng-boosted",
     "publish-lib": "npm run build-lib && npm run copy-readme && cd dist/ng-boosted && npm publish"
   },
-  "dependencies": {},
+  "dependencies": {
+    "eslint": "^6.7.2",
+    "focus-visible": "^5.0.2",
+    "serialize-javascript": "^2.1.2",
+    "set-value": "^3.0.1",
+    "tablesorter": "^2.31.2"
```

Comment:
I think there should only be `focus-visible` and `tablesorter` in `dependecies`, other should probably be in `devDependencies`. @Lausselloic any opinion?

Reply:
I think that tablesorter is not an angular library and use Jquery so can't be set as a dependency of ng-boosted.
focus-visible looks to be JS framework agnostic so why not and of course others looks to be dev-dependencies